policy/resolver.go

```diff
@@ -55,22 +61,24 @@ func (s *LocalServices) Assign(ctx context.Context, assignment *PolicyAssignment
 		}
 	}
 
-	// assign policy locally
-	deltas := map[string]*PolicyDelta{}
+	s.DataLake.EnsureAsset(ctx, assignment.AssetMrn)
+
+	policyActions := map[string]explorer.AssignmentDelta_Action{}
 	for i := range assignment.PolicyMrns {
 		policyMrn := assignment.PolicyMrns[i]
-		deltas[policyMrn] = &PolicyDelta{
-			PolicyMrn: policyMrn,
-			Action:    PolicyDelta_ADD,
-		}
+		policyActions[policyMrn] = explorer.AssignmentDelta_ADD
 	}
 
-	s.DataLake.EnsureAsset(ctx, assignment.AssetMrn)
+	frameworkActions := map[string]explorer.AssignmentDelta_Action{}
+	for i := range assignment.FrameworkMrns {
+		frameworkMrn := assignment.FrameworkMrns[i]
+		frameworkActions[frameworkMrn] = explorer.AssignmentDelta_ADD
+	}
 
-	_, err := s.DataLake.MutatePolicy(ctx, &PolicyMutationDelta{
-		PolicyMrn:    assignment.AssetMrn,
-		PolicyDeltas: deltas,
-		Action:       assignment.Action,
```

Comment:
I think, we need to get this back somehow. Before this PR, it could have had the values `explorer.Action_IGNORE` or `explorer.Action_ACTIVATE`.